src/System.Net.Requests/tests/FtpWebRequestTest.cs

```diff
@@ -70,7 +70,7 @@ public void GetResponse_ServerNameNotInDns_ThrowsWebException()
         public void GetResponse_ConnectFailure_ThrowsWebException()
         {
             // This is an invalid IP address, so we should fail to connect.
-            string serverUrl = "ftp://192.0.2.1/";
+            string serverUrl = "ftp://127.0.0.1/";
```

Comment:
I wouldn't recommend `::` in this case either.

You can try `ftp://serverthatshouldnotexist/" or similar.

The best solution would be to create a `Configuration.Ftp.cs` and add an entry there - if we do find in the future that 
doesn't somehow interfere with some existing address, we at least have a single place to change. (Just last week I found out that "doesntexist.com/" exists...)